clientv3/client.go

```diff
@@ -56,6 +57,8 @@ type Client struct {
 	ctx    context.Context
 	cancel context.CancelFunc
 
+	lastAllEps map[string]struct{}
```

Comment:
Judging by the grpc lb logic, it doesn't seem necessary to keep track of this since grpc already does some work to compute the delta between the old and new address lists. If it is needed, it probably belongs in the balancer code.